kube/m3dbnode-statefulset.yaml

```diff
@@ -82,7 +82,7 @@ spec:
             weight: 10
       containers:
       - name: m3db
-        image: quay.io/m3/m3dbnode:latest
+        image: quay.io/m3db/m3dbnode:latest
```

Comment:
could we use the `m3` repo for quay?

Reply:
Ah yeah I'll get #1364 landed first and then land this. For a while `/m3` was stale

Reply:
👍